### PR TITLE
fixed buttons getting outline focus when user moused out of button

### PR DESCRIPTION
**Pull request by noreenwu**
Changed the order of :focus to be after :hover and :active and that fixed the problem.  I have not gone through Design to see if they would want a different solution than what I have here. 
@heather @MagdalenaHenke

### CODE REVIEW

**Comment by noreenwu** [comment]
https://app.clubhouse.io/blocks/story/43892/buttons-hover-state-is-overriding-focus-state

**Comment by MagdalenaHenke** [comment]
This is another one...when I run `dev:docs` (or `gulp build` from within the blocks-base folder) on this branch, I get changes to dist/blocks.css, so I wonder whether that's the weird issue we had before or whether it's just an oopsie of not regenerating dist.

**Comment by noreenwu** [comment]
I think it might be specific to that local repository on my laptop at work. Everything I worked on in that repository did not update blocks.css and therefore I did not get nudged to commit it to git. So maybe I should abandon that local repo?

**Comment by MagdalenaHenke** [comment]
I'm still unclear about why those changes don't happen on your branch. You said, you tried something in a new repo and there it would cause blocks.css changes? 

:D whatever needs to be done, but we should make sure blocks.css in dist is up to date when this gets merged.

**Comment by MagdalenaHenke** [comment]
Second question: I think you mentioned going through other components with the same issue - e.g. checkboxes. Did you want to tackle those, too? 

We could either do it on this branch, or - being in favor of smaller PRs as I am! - on a separate one - then I'd just love to ticket the other "work" we're intending to do (e.g. fix this also for checkboxes and stuff).

File: packages/blocks-base/styles/buttons.styl [hunk 1]
```diff
@@ -162,6 +147,23 @@ $button-dark-active-text-color = $secondary-00;
       content: none;
     }
   }
+
+  &:focus, &.blx-focus {
```
[the remark this answers is not in the document]
Comment (noreenwu) [reply packages/blocks-base/styles/buttons.styl]: @MagdalenaHenke Thanks! Great find! I changed the order of hover, focus, active and active is now showing up. Would you review again? I will add the other components to Clubhouse.

**Comment by MagdalenaHenke** [comment]
Looks good to me! 🌮

**Pull request merged**